values.yaml.tmpl

```diff
@@ -2,15 +2,20 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-kotsadm:
-  kotsadmPassword: ""
+adminConsole:
+  password: ""
   automation:
     license:
       slug: ""
       data: ""
     skipPreflights: false
     config:
       values: ""
+    images:
+      kotsadm: kotsadm/kotsadm:${KOTS_TAG}
+      migrations: kotsadm/kotsadm-migrations:${KOTS_TAG}
+      minio: minio/minio:${MINIO_TAG}
```

Comment:
where do `MINIO_TAG` and `POSTGRES_ALPINE_TAG` get set?

Reply:
look for `curl -O -L https://raw.githubusercontent.com/replicatedhq/kots/${KOTS_TAG}/.image.env`

They are in that file

Reply:
ah great thanks